core/types/block.go

```diff
@@ -120,6 +120,33 @@ func (h *Header) HashNoNonce() common.Hash {
 	})
 }
 
+// HashNoPoASig returns the hash which is used as input for the proof-of-authority
```

Comment:
No. Felix made me split my clique PR up even though I told him it's getting messy since it's hard to find all the intermingled code snippets that are only needed by clique. I made quite a few "updates" already to move snippets out that Zsolt and others found.

I'm reluctant to rebase all 3 of my PRs once again to move this to the second commit. Can do if really necessary.